bdd/spec/023_http_spec.sh

```diff
@@ -30,4 +30,34 @@ Describe "@std/http"
       The output should eq "export const comeGetMe = \"You got me!\""
     End
   End
+
+  Describe "basic post"
+    before() {
+      sourceToAll "
+        from @std/app import start, print, exit
+        from @std/http import post
+
+        on start {
+          print(post('https://reqbin.com/echo/post/json', '{"test":"test"}').getOr('default'))
```

Comment:
Two things here:

1. You need to escape the `"` inside of that JSON with backslashes.
2. You don't need the `.getOr('default')` part, again, because `print` can handle `Result<Stringifiable>` types.